### PR TITLE
fix: move all init to the same file

### PR DESCRIPTION
**Pull request by markusahlstrand**
Noticed that the language config was a bit spread out, so moved it all into the utils/i18n file.

Looked a bit a the i18next library and it seems to be doing a few interesting things.. Maybe we should try to migrate all code to use this rather than our homebrewed solution?

### CODE REVIEW

**Comment by markusahlstrand** [comment]
Changed the default language back to swedish.. Think it should really be english now, but might require some changes to the snapshots?

File: src/constants.ts [hunk 1]
```diff
@@ -18,6 +18,6 @@ export const headers = {
 
 export const UNIVERSAL_AUTH_SESSION_EXPIRES_IN_SECONDS = 60 * 60 * 24; // 1 day
 
-export const CODE_EXPIRATION_TIME = 24 * 60 * 60 * 1000;
+export const CODE_EXPIRATION_TIME = 30 * 60 * 1000;
```
Comment (markusahlstrand) [inline src/constants.ts:21]: This matched what we were using in other places

File: src/routes/oauth2/callback.ts [hunk 2]
```diff
@@ -5,23 +5,6 @@ import { getClient } from "../../services/clients";
 import { HTTPException } from "hono/http-exception";
 import { OpenAPIHono, createRoute, z } from "@hono/zod-openapi";
 import { Var } from "../../types/Var";
-import i18next from "i18next";
```
Comment (markusahlstrand) [inline src/routes/oauth2/callback.ts:8]: Don't think it is used here? Don't really like how it's designed with a singleton. What if you have multiple simultaneous requests? Feels like a recipe for really nasty bugs.

File: src/utils/i18n.ts [hunk 3]
```diff
@@ -17,11 +18,32 @@ const labels: Labels = {
...
+  if (!labels[language]) {
+    throw new Error(`Language ${language} is not supported`);
+  }
+
+  return labels[language];
+}
+
+export function initI18n(lanaguage = "sv") {
+  i18next.init({
+    lng: lanaguage,
```
Comment (markusahlstrand) [inline src/utils/i18n.ts:40]: I actually don't think it's intended to be used like this.. This seems to be and escape hatch to override the language detection, so I think we really should move the init to the process start and use the language detection instead?

**Comment by danielgent** [comment]
Do whatever you want Markus but the tests need to actually pass 🫡

File: src/routes/oauth2/callback.ts (hunk 2 above)
Comment (danielgent) [reply src/routes/oauth2/callback.ts:8]: It definitely is used here I did this last week. If you get the tests passing first then you'll see what I mean

I would close this PR. what we have is already working. If you want to make a change then do one at once etc etc green checks first. Coding is easy, testing is the hard part

**Comment by markusahlstrand** [comment]
Think I'll do a new PR with this.. Had a look on the code again and think we should switch to use i18next in all places, also for the email templates.
It is possible to pass the language when fetching an actual translation which in a way is safer, but.. it's more code and more things to pass around. Assuming the that the code is not async the current solution would work, but it requires you to be suuuuper picky with how you use it. As soon as there's something async there's a risk that we'll be using the wrong language.

**Pull request closed without merging**